admin/app/adapters/complementary-certification.js

```diff
@@ -11,16 +11,17 @@ export default class ComplementaryCertificationAdapter extends ApplicationAdapte
     return `${this.host}/${this.namespace}/complementary-certifications/${id}/badges`;
   }
 
-  async updateRecord(store, type, complementaryCertification) {
+  async updateRecord(_, type, complementaryCertification) {
```

Comment:
Pinaillage

Pas à voir avec le changement effectué, mais je remarque que c'est le seul updateRecord qui a modifié la signature de base pour mettre `complementaryCertification` au lieu de `snapshot` (alors qu'au final cet objet ne représente pas ce model) 🤔 

```suggestion
  async updateRecord(_, type, snapshot) {
```